py4web/core.py

```diff
@@ -591,6 +591,9 @@ def URL(
     URL('a','b',vars=dict(x=1),scheme='https') -> https://{domain}/{script_name?}/{app_name}/a/b?x=1
     URL('a','b',vars=dict(x=1),use_appname=False) -> /{script_name?}/a/b?x=1
     """
+    if use_appname is None:
+        use_appname = request.headers.get("x-py4web-appname")
+        use_appname = True if use_appname is None else not use_appname
```

Comment:
Do not understand this. "x-py4web-appname": "false" would evaluate to true?

Reply:
Yes, it looks horrible and would be better to have `proxy_set_header x-py4web-use-appname !$appname`, but nginx doesnt support these things and it requires if-block + yet another var

Reply:
Just in case:
https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/